### PR TITLE
All timeout/interval functions are replaced with hammerhead.browserNatives analogues (closes #431)

### LINKED ISSUE #431: Replace all timeout/interval function with hammerhead.browserNatives analogues
We can completely implement this when https://github.com/DevExpress/testcafe-hammerhead/issues/519 is implemented and published

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin @georgiy-abbasov

### CODE REVIEW

**Comment by inikulin** [comment]
Seems like it would be nice if our eslint plugin handle this cases \cc @LavrovArtem

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: src/client/ui/progress-panel/index.js [hunk 1]
```diff
@@ -152,23 +153,23 @@ export default class ProgressPanel {
...
-            window.clearInterval(this.updateInterval);
+            nativeMethods.clearInterval.call(window, this.updateInterval);
             this.updateInterval = null;
         }
 
         if (success) {
             if (this.startTime && Date.now() - this.startTime < MIN_SHOWING_TIME) {
-                window.setTimeout(() => {
+                nativeMethods.setTimeout.call(window, () => {
                     window.setTimeout(() => this._hidePanel(false), SHOWING_DELAY);
```
Comment (georgiy-abbasov) [inline src/client/ui/progress-panel/index.js]: I think, we should replace this expression too

Comment (helen-dikareva) [inline src/client/ui/progress-panel/index.js]: Yep, so we really need auto check

**Comment by inikulin** [comment]
ping @LavrovArtem

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@inikulin I'm working with the eslint plugin

**Comment by LavrovArtem** [comment]
Update a version of the `eslint-plugin-hammerhead` to `0.0.6` and add the rule in `src\client\.eslintrc`:
`"hammerhead/use-native-methods": [2, ["setTimeout", "setInterval", "clearTimeout", "clearInterval"]]`

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

**Comment by georgiy-abbasov** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**